### ISSUE TITLE
Reverse Bold Fonts Admin Page

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1260]
I suggest we bolden the font on the "Forces" collapsable tabs of the admin page and put the normal font on the title of the tab inside the collapsable:
![Reverse Fonts](https://user-images.githubusercontent.com/85491794/145838558-bdd8385e-0762-45c9-b123-114aa5a9b285.png)

**Comment by ShendMikullovci** [comment]
Same with "Roles" and "Assets".

**Comment by ShendMikullovci** [comment]
Question: What is the purpose of the Umpire toggle?

**Comment by IanMayo** [comment]
1. This really looks like a couple of instances of duplication.  I don't think we need both labels.
2. The `Umpire` toggle indicates that members of this force are to be treated as umpires.  So, they are shown the `Private Message` box for all messages they send.

How about we remove the second instance of each title, @ShendMikullovci ?

**Comment by ShendMikullovci** [comment]
Good idea. Here's how that would look:

![Admin Page Forces UX (2)](https://user-images.githubusercontent.com/85491794/146361917-0051f147-b6e8-473e-a42e-eea8b887df7d.png)

I also took the Umpire toggle outside of the Objective & Overview tab. It's a highly important feature that needs to remain in working memory and be visible while the user is going through different tabs in the Forces window.

Here's the collapsed tab with the header removed:

![Admin Page Forces UX Collapsed Final](https://user-images.githubusercontent.com/85491794/146362139-a9feb114-2cc9-463f-b9b0-7e94efe7c684.png)
 
Let me know your thoughts.

**Comment by IanMayo** [comment]
Sure, that's all good, thanks @ShendMikullovci 

I suspect we should probably have some kind of "more info" icon by the `Umpire` toggle. In the tooltip it would indicate that this setting this toggle means that players of this force are able to read & write `Private` message fields.

**Comment by ShendMikullovci** [comment]
Good idea. I was thinking of a small question mark next to the toggle.
Here's how that could look: 

![Question Mark Feature](https://user-images.githubusercontent.com/85491794/146371383-365b0cb6-1507-4cee-a320-4a2fa509642e.png)

Clicking on it would open a small window explaining what the toggle is about: 

![Question Mark Explained](https://user-images.githubusercontent.com/85491794/146372219-8b052db9-28d2-4f73-ba0a-b27f34698cd8.png)

**Comment by IanMayo** [comment]
I may be mistaken, but I think the _standard_ place is to the right of the label.  I'll Google it a little later.
